docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -478,6 +541,54 @@ The treatment algorithm is described in this :download:`word document found here
 
 **How to assign treatment effects:**
 
+Each treatment category has a hazard ratio associated with it both for progression-free survival and overall survival relative to the overall progression-free survival and overall survival of their demographic group as a whole. There are separate hazard ratios for the first line of treatment and later lines of treatment for each treatment category. Additionally, the hazard ratios for the isatuxamib- and daratumumab-containing treatment categories vary based on retreatment status (:code:`retreatment == True`). The hazard ratios are shown in the tables below.
+
+.. list-table:: First line of treatment hazard ratios
+   :header-rows: 1
+
+   * - Treatment category
+     - Progression free survival HR (95% UI)
+     - Overall survival HR (95% UI)
+   * - All categories combined
+     - 1 (reference)
+     - 1 (reference)
+   * - Isatuxamib-containing treatment category
+     - 0.429 (0.368, 0.495)
+     - 0.760 (0.645, 0.895)
+   * - Daratumumab-containing treatment category
+     - 0.429 (0.368, 0.495)
+     - 0.760 (0.645, 0.895)
+   * - Residual treatment category
+     - 1.00581 (1.0051, 1.0064)
+     - 1.0024 (1.0011, 1.0036)
+
+.. list-table:: Second and later lines of treatment hazard ratios
+   :header-rows: 1
+
+   * - Treatment category
+     - Progression free survival HR (95% UI)
+     - Overall survival HR (95% UI)
+   * - All categories combined
+     - 1 (reference)
+     - 1 (reference)
+   * - Isatuxamib-containing treatment category, :code:`retreated != True`
+     - 0.530 (0.356, 0.803)
+     - 1.159 (1.044, 1.185)
+   * - Isatuxamib-containing treatment category, :code:`retreated == True`
+     - 0.765 (0.678, 0.902)
+     - 1.318 (1.088, 1.37)
+   * - Daratumumab-containing treatment category, :code:`retreated != True`
+     - 0.217 (0.203, 0.231)
```

Comment:
I am surprised that this is so low

Reply:
Unfortunately, I think this may be because of the R (lenalidomide) in dara-Rd, which is a pretty effective drug. 

There is clinical trial data for dara-pomalidomide-dexamethasone, but it is not approved by the FDA for this application (trial data just came out in 2021). I think dara-Pd would be a reasonable selection instead of dara-Rd because the isatuxamib in the RR stetting is in combination with pomalidomide and dexamethasone and also, lenalidomide is one of the most commonly used drugs in early lines of treatment and pomalidomide is recommended for use after lenalidomide, so I think it is reasonable. 

I can test and see what the HRs would be with the dara-Pd regimen instead of dara-Rd, although I originally chose not to use dara-Pd due to the lack of current FDA approval.

Reply:
The dara-Pd PFS HR = 0.66, which is pretty much equivalent to the isa-Pd PFS HR (0.67) relative to flatiron, as expected.

The dara-Pd OS HR is 1.03, which is slightly favored over isa-Pd of 1.16 relative to flatiron.

We could also pull in dara-carfilzomib-dexamethasone (dara-Kd) data, which we have done for isatuxamib (the recently published IKEMA trial), so that we are using the same combination regimens between isa/dara. Dara-Kd is approved for use by the FDA. The dara-Kd regiment has very low HRs (0.1 PFS, 0.67 OS)